src/System.Globalization/tests/CultureInfo/CultureInfoParent.cs

```diff
@@ -2,40 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Globalization;
 using Xunit;
 
 namespace System.Globalization.Tests
 {
     public class CultureInfoParent
     {
-        [Fact]
-        public void PosTest1()
-        {
-            CultureInfo myExpectParentCulture = new CultureInfo("en");
-            CultureInfo myTestCulture = new CultureInfo("en-us");
-            Assert.Equal(myTestCulture.Parent, myExpectParentCulture);
-        }
-
-        [Fact]
-        public void PosTest2()
-        {
-            CultureInfo myTestCulture = new CultureInfo("en");
-            CultureInfo myExpectParent = CultureInfo.InvariantCulture;
-            Assert.Equal(myTestCulture.Parent, myExpectParent);
-        }
-
-        [Fact]
-        public void PosTest3()
+        [Theory]
+        [InlineData("en-US", "en")]
+        [InlineData("en", "")]
+        [InlineData("", "")]
+        public void Parent(string name, string expectedParentName)
         {
-            CultureInfo myExpectParent1 = new CultureInfo("");
-            CultureInfo myTestCulture = CultureInfo.InvariantCulture;
-            Assert.Equal(myTestCulture.Parent, myExpectParent1);
+            CultureInfo culture = new CultureInfo(name);
+            Assert.Equal(culture.Parent, new CultureInfo(expectedParentName));
         }
 
         [Fact]
-        public void PosTest4()
+        public void Parent_MultipleParents()
```

Comment:
"ParentChain" or "Ancestors" or something like that might be better than "MultipleParents", which to me suggests that a single entry has multiple parents at the next level, rather than this being about the parent of the parent, etc.